### PR TITLE
add player_version mp command

### PR DESCRIPTION
**Pull request by gfgtdf**
to tell you what wesnoth version another player is using.

### CODE REVIEW

**Comment by ProditorMagnus** [comment]
I suggest "version" instead of "player_version" to use command.

**Comment by CelticMinstrel** [comment]
With `:version <player>` I don't think there's a need to include the word "player" in the command name.

**Comment by gfgtdf** [comment]
i names it player_version to make it clear that it doesn't tell you the version of the server, (which is what i'd expect form a command that is called 'version'), maybe i'll make it so that it tells you the server version number when version is used with no argument.

**Comment by ProditorMagnus** [comment]
Is game_config::version enough for server version?

**Comment by gfgtdf** [comment]
> Is game_config::version enough for server version?

the commandline --version uses it aswell so i guess that's as good as it gets.

File: src/server/server.cpp [hunk 1]
```diff
@@ -1081,7 +1081,7 @@ void server::handle_query(socket_ptr socket, simple_wml::node& query)
 		command == "motd" ||
 		command == "netstats" ||
 		command == "netstats all" ||
-		command == "player_version" ||
+		command.compare(0, 7, "version") == 0 ||
```
Comment (CelticMinstrel) [inline src/server/server.cpp]: Why are you using `.compare` here?

File: src/server/server.cpp [hunk 2]
```diff
@@ -2296,7 +2296,13 @@ void server::player_version_handler(
 			return;
 		}
 	}
-	*out << "Player " << parameters << " not found.";
+	if(parameters.empty()) {
```
Comment (CelticMinstrel) [inline src/server/server.cpp]: Wouldn't it make more sense to test for empty _before_ looping over all the players?

File: src/server/server.cpp (hunk 1 above)
Comment (gfgtdf) [reply src/server/server.cpp]: becasue the request is not "version" but "version <playername>"

File: src/server/server.cpp (hunk 2 above)
Comment (gfgtdf) [reply src/server/server.cpp]: it could but it wouldn't change behviour.

File: src/server/server.cpp (hunk 1 above)
Comment (CelticMinstrel) [reply src/server/server.cpp]: But this'll also match anything nonsensical beginning with "version".
